.travis.yml

```diff
@@ -47,10 +45,7 @@ before_script:
   - 'sleep 20'
 
 script:
-  - ng build # this does syntax checking, unit testing, and jshint
-  - ng serve --silent &
-  - ng test --watch=false --code-coverage
-  - cypress run --record --config defaultCommandTimeout=10000 #test using cypress for integration testing
+  - ./scripts/run-prod.sh
```

Comment:
script name is a bit misleading, since it only runs prod half of the time
maybe `run-build.sh`?